frontend/simple/views/Contributions.vue

```diff
@@ -1,69 +1,78 @@
-            :isMonetary="isMonetary(contribution.type)"
-            @interaction="showReceivingSettings"
-          >
-            <span v-html="textReceivingContribution(contribution, index)"></span>
-            <template v-if="hasWhoElse(contribution.who)">
-              <i18n>and</i18n>
-              <tooltip>
-                <button class="gi-is-unstyled gi-is-link-inherit">{{contribution.who.length - 1}}<i18n>others</i18n></button>
-                <template slot="tooltip">
-                  <p v-for="name, index in contribution.who" v-if="index > 0">{{name}}</p>
-                </template>
-              </tooltip>
-            </template>
+          <contribution v-for="contribution in receiving.nonMonetary">
+            <span v-html="textReceivingNonMonetary(contribution)"></span>
+            <TextWho :who="contribution.who"></TextWho>
           </contribution>
+
+          <trigger @animate="updateSize">
+            <contribution v-if="doesReceiveMonetary" variant="editable" isMonetary @interaction="handleFormTriggerClick">
+              <span v-html="textReceivingMonetary(receiving.monetary)"></span>
+              <TextWho :who="groupMembersPledging"></TextWho>
+              <i18n>each month</i18n>
+            </contribution>
+          </trigger>
         </ul>
-      </section>
+      </div>
 
-      <section class="column">
+      <div class="column">
         <i18n tag="h2" class="title is-3">Giving</i18n>
 
-        <ul class="c-ul" v-if="givesNonMonetary || givesMonetary">
+        <ul class="c-ul">
           <contribution v-for="contribution, index in giving.nonMonetary"
             class="has-text-weight-bold"
             variant="editable"
             @new-value="(value) => handleEditNonMonetary(value, index)"
           >{{contribution}}</contribution>
 
-          <contribution v-if="givesMonetary"
-            variant="editable"
-            isMonetary
-            @interaction="showGivingMincomeSettings"
-          >
-            <span class="has-text-weight-bold">{{currency}}{{giving.monetary}}</span> <i18n>to other's mincome</i18n>
-          </contribution>
-        </ul>
-
-        <ul class="c-ul">
           <contribution variant="unfilled" @new-value="submitAddNonMonetary">
             <i class="fa fa-heart c-contribution-icon" aria-hidden="true"></i>
             <i18n>Add a non-monetary method</i18n>
           </contribution>
 
-          <contribution v-if="!givesMonetary" variant="unfilled" isMonetary @interaction="addMonetaryMethod">
-            <i class="fa fa-money c-contribution-icon" aria-hidden="true"></i>
-            <i18n>Add a monetary method</i18n>
-          </contribution>
+          <trigger @animate="updateSize">
+            <contribution v-if="doesGiveMonetary" variant="editable" isMonetary @interaction="handleFormTriggerClick">
+              <!-- REVIEW - have different text if the user is pledging $0 -->
```

Comment:
@sandrina-p I agree we should have a different message... but I'm also wondering when entering Income Details if it should have a checkbox that says something like "Pledging non-monetary item" and it should not allow you to save $0 as a pledge amount. What are your thoughts? @taoeffect @dotmacro

Reply:
The current UI on the form IMO is well done "how much can you Pledge to the group?". It makes sense the user to say 0$.

Then the created contribution box... could say: "The group average pledging is $50. You don't pledge any amount at the moment". I don't know... I believe @dotmacro will know better what to tell in this case.

Reply:
Having a $0 pledge on the contributions screen may help remind them to keep this info updated if something changes. 

Because the form won't let the user continue without entering a number (or canceling), I think it is okay to word $0 the same as all other pledge amounts. Maybe something like:  
"Pledging up to $n to others' mincome. (The group's average pledge is $ave.)"  
or we can leave off the average bit, and have that info only with the chart.

Reply:
Okay, edited, now it's like this:
![image](https://user-images.githubusercontent.com/14869087/46569430-cf744080-c94c-11e8-976f-49d75057459e.png).